### PR TITLE
Deployed dashboard to servermgr (craftassist.io heroku)

### PR DESCRIPTION
**Pull request by snyxan**
# Description

Deployed dashboard to craftassist.io so that minecraft client is not needed any more. Removed landing page and directly launched an ecr instance. When finished, automatically redirect to dashboard on port 8000. 

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [x] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change

## Type of requested review

- [ ] I want a thorough review of the implementation.
- [x] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

Now when you visit craftassist.io, it will directly launch a ecr instance show the progress page (this will further be removed when mephisto pool gets in).

Screen Shot 2021-02-18 at 6 47 55 PM

After the instance is fully launched, it will redirect to the dashboard page:

Screen Shot 2021-02-18 at 7 00 46 PM

# Testing

Tested locally.

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [x] I have added relevant collaborators to review the PR before merge.

### CODE REVIEW

File: craftassist/servermgr/app.py [hunk 1]
```diff
@@ -42,29 +40,9 @@
...
-    return flask.render_template("index.html")
-
-
 @app.route("/launch", methods=["GET", "POST"])
-# The limiter ensures that the web server will handle at most 60 requests of
-# each unique ip address or 100 requests in total per mintue to prevent
-# malicious attack. e.g. one user may make too many 'launch instances' requests
-# to flood ECS with containers
-@ratelimit(max_requests=60, seconds=60, key_func="ip")
-@ratelimit(max_requests=100, seconds=60, key_func="global")
```
Comment (kavyasrinet) [inline craftassist/servermgr/app.py:67]: Any reason we are removing the rate limiter ?

File: craftassist/servermgr/app.py [hunk 2]
```diff
@@ -149,54 +126,6 @@ def status():
     return json.dumps({"progress": 100, "ip": ip})
 
 
-@app.route("/matchmaker")
```
Comment (kavyasrinet) [inline craftassist/servermgr/app.py:152]: Can we keep this somewhere in arxiv ? This was useful to do some matchmaking human - human experiments.

File: craftassist/servermgr/app.py [hunk 3]
```diff
@@ -149,54 +126,6 @@ def status():
...
-    elif d["action"] == "wait":
-        while True:
-            instance_id = rconn.get(d["wait_key"])
-            if instance_id:
-                return flask.redirect("/wait/{}?role=assistant".format(urlencode(instance_id)))
-            else:
-                time.sleep(0.5)
-
-
-@app.route("/survey", methods=["POST"])
```
Comment (kavyasrinet) [inline craftassist/servermgr/app.py:189]: we probably should also arxiv this. At some point, we might want to re-introduce this ?

File: craftassist/servermgr/rate_limiter.py [hunk 4]
```diff
@@ -1,76 +0,0 @@
-"""
```
Comment (kavyasrinet) [inline craftassist/servermgr/rate_limiter.py:1]: same comment as before - can we arxiv / keep in internal somewhere perhaps ?

File: craftassist/servermgr/templates/index.html [hunk 5]
```diff
@@ -1,58 +0,0 @@
-{% extends 'base.html' %}
```
Comment (kavyasrinet) [inline craftassist/servermgr/templates/index.html:1]: Also Arxiv please :)

File: craftassist/servermgr/templates/matchmaker.html [hunk 6]
```diff
@@ -1,42 +0,0 @@
-{% extends 'base.html' %}
```
Comment (kavyasrinet) [inline craftassist/servermgr/templates/matchmaker.html:1]: Arxiv also please.

**Review by kavyasrinet — COMMENTED** [review]
LGTM! Thanks for setting this up.
A meta comment - can we please Arxiv the setup we used to have, somewhere ? Perhaps in internal ? A lot of the work on survey, matchmaker, launch_button, index `html` files should be Arxiv'd in case we want to reuse it later.

File: craftassist/servermgr/app.py (hunk 1 above)
Comment (snyxan) [reply craftassist/servermgr/app.py:67]: Initially it was removed for development purpose. Then I realized if we are doing container pool on Mephisto side, we might need to change the rate limiter for a bit. so removed it for now.

Don't have strong opinion here, can add it back if y'all want.

**Comment by snyxan** [comment]
> LGTM! Thanks for setting this up.
> A meta comment - can we please Arxiv the setup we used to have, somewhere ? Perhaps in internal ? A lot of the work on survey, matchmaker, launch_button, index `html` files should be Arxiv'd in case we want to reuse it later.

Create a wiki page to save all archived work. https://www.internalfb.com/intern/wiki/Droidlet/Archive/Old_Servermgr_setup/

**Review by kavyasrinet — APPROVED** [review]
Thanks for creating the Arxiv!

**Pull request merged**